### PR TITLE
Change net api fields to more logical names

### PR DESCRIPTION
**Pull request by yanirq**
Spec.net field names changes:
vendor => vendorId
device => deviceId

### CODE REVIEW

File: api/v2/performanceprofile_types.go [hunk 1]
```diff
@@ -144,10 +144,10 @@ type Device struct {
 	InterfaceName *string `json:"interfaceName,omitempty"`
 	// Network device vendor ID represnted as a 16 bit Hexmadecimal number.
 	// +optional
-	VendorID *string `json:"vendor,omitempty"`
+	VendorID *string `json:"vendorId,omitempty"`
```
Comment (fromanirh) [inline api/v2/performanceprofile_types.go:147]: should be `vendorId` or `vendorID` (lowercase or uppercase 'D') ?
Here and in all the following instances.
Everything else lgtm

Comment (yanirq) [reply api/v2/performanceprofile_types.go:147]: Actually I did'nt stumble upon a good guideline for this (especially in our code)  so I decided to go with a Capital separator. 
I'm fine either way

Comment (fromanirh) [reply api/v2/performanceprofile_types.go:147]: > Actually I did'nt stumble upon a good guideline for this (especially in our code) so I decided to go with a Capital separator.
> I'm fine either way

Pretty much the same here. I'd say:
- let's be consistent with our current API - if relevant (can't remember if we have already someIds)
- let's just quickly review what's the latest standard on ocp/k8s, and file an issue should we change the code.

Comment (yanirq) [reply api/v2/performanceprofile_types.go:147]: looking at some openshift/k8s projects it seems they all comply to using ID/UID in capitals. changing accordingly

**Review by fromanirh — COMMENTED** [review]
/lgtm

**Pull request merged**